package.json

```diff
@@ -5,7 +5,7 @@
   "main": "listener.js",
   "scripts": {
     "deploy-development": "./node_modules/.bin/node-lambda deploy -e development -f ./config/development.env -o arn:aws:iam::224280085904:role/lambda_basic_execution -S ./config/event_sources_development.json -a $AWS_ACCESS_KEY_ID_DEVELOPMENT -s $AWS_SECRET_ACCESS_KEY_DEVELOPMENT",
-    "deploy-qa": "./node_modules/.bin/node-lambda deploy -e qa -f ./config/qa.env -o arn:aws:iam::946183545209:role/lambda-full-access -S ./config/event_sources_qa.json -a $AWS_ACCESS_KEY_ID_QA -s $AWS_SECRET_ACCESS_KEY_QA",
+    "deploy-qa": "./node_modules/.bin/node-lambda deploy -e qa -f ./config/qa.env -o arn:aws:iam::946183545209:role/lambda-full-access -S ./config/event_sources_qa.json -a $AWS_ACCESS_KEY_ID_QA -s $AWS_SECRET_ACCESS_KEY_QA --layers arn:aws:lambda:us-east-1:946183545209:layer:lib-for-node10-wrapped-php7:1",
```

Comment:
You might want to get devops to update the name for the role so it doesn't include `node10`..